wri/gfw
# [FLAG-920] Update all VIIRS tables version

## Overview

We need to update the VIIRS table versions to fix the alert count issues that were resolved in https://gfw.atlassian.net/browse/GTC-2466 . [According to Justin](https://gfw.atlassian.net/browse/GTC-2466?focusedCommentId=20325), “all versions for VIIRS tables (gadm/geostore/WDPA) can be set to v20230901. Let me know if you'd llike me to explicitly list out the tables."

table | version
-- | --
gadm__viirs__iso_weekly_alerts | v20230901
gadm__viirs__adm1_weekly_alerts | v20230901
gadm__viirs__adm2_weekly_alerts | v20230901
gadm__viirs__adm2_daily_alerts | v20230901
gadm__viirs__iso_whitelist | v20230901
gadm__viirs__adm1_whitelist | v20230901
gadm__viirs__adm2_whitelist | v20230901
geostore__viirs__weekly_alerts | v20230901
geostore__viirs__daily_alerts | v20230901
geostore__viirs__whitelist | v20230901
wdpa_protected_areas__viirs__weekly_alerts | v20230901
wdpa_protected_areas__viirs__daily_alerts | v20230901
wdpa_protected_areas__viirs__whitelist | v20230901


## Demo
![Screenshot 2023-09-15 at 11 47 32](https://github.com/wri/gfw/assets/23243868/dde2ae89-6b9b-4b6b-8ae0-df4267bd6906)

## Notes

Each table has an object property inside `analysis-datasets-versions.json` to set version. Eg. 

``` 
// gadm__viirs__iso_weekly_alerts
VIIRS_ADM0_WEEKLY = v20230901
```

## Testing

- Open the dashboard
- Check fire alerts widgets (You need to rollback #4678 to get VIIRS instead Modis)
- Validate DATA API version response
